src/triagebot/pr-assignment.md

```diff
@@ -74,6 +74,16 @@ fallback = ["@Mark-Simulacrum"]
 If the `owners` map is configured, then triagebot will automatically select a reviewer based on which files were modified in the PR.
 The existence of the owners table also enables the ability for users to post a comment with `r? name` to set the assignment to a specific user.
 
+### Vacation
+
+If a reviewer wants to temporarily prevent themselves from being auto-assigned they can add themselves to the special
```

Comment:
```suggestion
If a reviewer wants to temporarily prevent themselves from being assigned (automatically or manually) they can add themselves to the special
```

or something like that. I'm pretty sure that even direct/manual assignment are prevented (e.g., `r? @USER` in a PR description and/or comment)

Reply:
See https://github.com/rust-lang/rust/pull/120177 for example